src/patterns/step-by-step-navigation/index.md

```diff
@@ -101,12 +101,16 @@ If there is not a set order, they should be listed in an order that helps the us
 
 For example, ‘Agree a contract and salary’ is listed as step 4 in ‘Employ someone: step by step’ although it can be completed earlier.
 
+Ensure the number of the step is read out to screen-reader users when navigating to information within the `button` element.
+
 ### Use both ‘and’ and ‘or’
 
 Use ‘and’ to show when you can or must complete steps at the same time. You can use ‘and’ for more than 2 steps.
 
 Use ‘or’ to show when there’s a choice between 2 steps, or when a task must be completed in a different way based the user’s circumstances or eligibility.
 
+When using ‘and’ or ‘or’ nest the non-sequential steps as sub-headings of the step they relate to as level 3 headings in the list. This will convey that the ‘and’ or ‘or’ steps are not the next steps in the sequential order for screen reader users.
```

Comment:
```suggestion
When using ‘and’ or ‘or’, make sure screen reader users can easily tell the difference between additional information or actions  and the order of the steps. Do this by nesting non-sequential steps as sub-headings within the related step.
```

Reply:
testing whether turning the sentence around makes it clearer or not.

Reply:
Hmm... it feels like the heading nesting might be a clearer concept to peg this explanation to?

My best go at it:
```suggestion
When adding `and` or `or` steps, tag them as sub-headings (usually `<h3>`) under the numbered step they're placed after. This is to show to screen reader users that these related steps should be considered as a whole, rather than one-by-one.
```

Reply:
Definitely feel there’s something in both approaches. I like changing the sentence around.

Reply:
I've had another go @calvin-lau-sig7 @claireashworth , what do you think?